nideep/datasets/amfed/amfed.py

```diff
@@ -138,9 +138,8 @@ def __init__(self, dir_prefix, video_type=VIDEO_TYPE_FLV, cache_dir=None):
         Constructor
         '''
         self.logger = logging.getLogger(__name__)
-        logging.basicConfig(level=logging.INFO)
         self.cache_dir = cache_dir
-        if not os.path.exists(cache_dir):
+        if cache_dir and not os.path.exists(cache_dir):
             os.makedirs(cache_dir)
```

Comment:
print that cache director is being created and its **absolute path** using logging.info()